core/toolbox/category.js

```diff
@@ -389,7 +415,7 @@ Blockly.ToolboxCategory.prototype.addColourBorder_ = function(colour) {
  * @protected
  */
 Blockly.ToolboxCategory.prototype.getColour_ = function(categoryDef) {
-  var styleName = categoryDef['categorystyle'];
+  var styleName = categoryDef['categorystyle'] || categoryDef['categoryStyle'];
```

Comment:
Why is the order of check different here than above? lowercase then uppercase, vs uppercase then lowercase.

Reply:
No good reason. Changed them all to be the same.